### DISCUSSION TITLE
ArrayCards组件中，怎么给ArrayCards.Addition自定义默认值呢？

### DISCUSSION THREAD

**Discussion by feixia5320** [alibaba/formily#1560] [Q&A 1.x] [answered]
ArrayCards组件中，点击增加数据ArrayCards.Addition时，怎么自定义默认值呢？

**Comment by janryWang** [comment, marked as answer]
props上有defaultValue属性

**Reply by feixia5320** [reply]
ArrayCards.Addition这个组件只有 title和method属性，props中，也没有defaultvalue

**Reply by janryWang** [reply]
类型签名和文档少加了，其实是支持的

**Reply by jonny-gg** [reply]
defaultValue 有什么示例吗

**Reply by jonny-gg** [reply]
看源码似乎是clone里面东西，但是clone里面格式还挺乱，没看懂
